utils/forward-merge.js

```diff
@@ -285,7 +287,7 @@ function parseContents(lines) {
  * @param file {string}
  */
 async function resolveJsonFile(file) {
-  const contents = await fs.readFile(file).toString();
+  const contents = (await fs.readFile(file)).toString();
```

Comment:
Without this fix, it will `toString` the promise instead of the result of the promise. The `package.json` files were getting replaced with `[object Promise]` and that won't do.